unit_1d_hw.py

```diff
@@ -7,6 +7,128 @@
 '''
 import random # We will talk about imports later :)
 
+'''
+DEV NOTES:
+
+For Unit 1D They learn te below:
+
+- Functions 
+    - Creating a function
+    - Returning a value (including void)
+    - Creating parameters for function
+    - Default arguments
+    - Multiple returns
+
+- Classes
+    - calling .append on a list (may need to use a fixture for this)
+    - Creating an object 
+    - Creating the init function
+    - Creating more functions in the same class
+
+
+Using built in functions
+- Range
+- Possibly others
+
+'''
+
+#### Create a Function Below! ####
+'''
+Create a function called function_1(), have it return None
+
+Hint: Syntax to create a function:
+
+def <function_name>():
+    return <some value>
+
+'''
+
+##################################
+
+#### Create a Function Below ####
+'''
+Create a function called function_return_1() that returns the integer 5
+
+'''
+
+##################################
+
+#### Create a Function Below ####
+'''
+Create a function called function_return_1() that returns the string "I am a string!"
+
+'''
+
+##################################
+
+#### Create a Function Below ####
+'''
+Create a function called add_one, that takes in a parameter x, adds 1 then returns that value
+
+'''
+
+##################################
+
+
+#### Create a Function Below ####
+'''
+Create a function called add_x_y, that takes in a parameter x, and a parameter y, adds x and y then returns that value
+
+'''
+
+##################################
+
+
+#### Create a Function Below ####
+'''
+Create a function called slope, that takes in parameters, m, x and b. Where:
+
+:param m: (int) The slope to scale the the list x
+:param x: (list) The list of values we want to scale and shift based on m and b
+:param b: (int) The intercept to shift the values of x
+
+Have the function scale each value of x then add b and store it back into x.
+
+Have the function return x
+
+Hint: Use a for loop to iterate through the list
+'''
+
+##################################
+
+def slope(m, x, b):
+    return x
+
+#### Create a Function Below ####
+'''
+Create a function called multi_return, that takes in parameters, x and y
+
+:param x: (int) Integer passed in
+:param y: (int) Integer passed in
+
+Have the function scale each value of x by 5 and add 2 to y. 
```

Comment:
I understand what you mean by `each` since you could call the function multiple times, but it also gives me images of the lists being passed in for `x` in the previous problems, so maybe just say scale the value of `x`?

```suggestion
Have the function scale the value of x by 5 and add 2 to y. 
```

Reply:
Thanks!